app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
- [ ] Please one more thing to do, kindly implement the method that updates the comments counter for a post as instructed in the project requirement. 🙏

Reply:
I don't know how I missed this one. Thanks for the constructive feedback @OyePriscilla.